docs-kits/kits/Industry Core Kit/Software Development View/part_aspect-model-overview.mdx

```diff
@@ -20,7 +20,7 @@ This work is licensed under the CC-BY-4.0 (https://creativecommons.org/licenses/
 - Source URL: https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/tree/main/docs-kits/kits/Industry%20Core%20Kit (latest version)
 -->
 
-In the following section, all aspect models that are part of Industry Core are documented. 
+The following section documents all aspect models that are part of or relate to an Industry Core digital twin.
 
 | Digital Twin Type | Aspect Model | Mandatory Version | Optional Versions | KIT |
```

Comment:
```suggestion
| Digital Twin Type | Aspect Model | Mandatory Version | Optional Versions | KIT |  Standardized |
```

In this case I would suggest to add an addition field within the table for "standardized" and assign the respective CX-Number or N/A to prevent wrong assumptions related to the actual standard.